File: PRAK605/PRAK605-2210817310015-RIDHANISETIADI.py
x = int(input())
MatriksA = []
print("Matriks A")
for i in range(x):
    k = list(map(int,input().split()))
    MatriksA.append(k)
MatriksB = []
print("Matriks B")
for i in range(x):
    k = list(map(int,input().split()))
    MatriksB.append(k)
MatriksAB = []
print("\nMatriks AXB")
for i in range(x):
    MatriksAB.append([])
    for j in range(x):
        a=0
        for k in range(x):
            a=a+MatriksA[i][k]* MatriksB[k][j]
        MatriksAB[i].append(a)
for baris in MatriksAB :
    for k in baris :
        print(k, end=' ')
    print()

File: PRAK605/PRAK605-2210817310015-RIDHANISETIADI.c
#include <stdio.h>
int main(void){
    int x, y, z, ordo, isi, hasil;
    scanf("%d", &ordo);
    int MatriksA[ordo][ordo],MatriksB[ordo][ordo];
    printf("Matriks A\n");
    for(x=0; x<ordo; x++){
        for(y=0; y<ordo; y++){
            scanf("%d", &isi);
            MatriksA[x][y]= isi;
        }
    }
    printf("Matriks B\n");
    for(x=0; x<ordo; x++){
        for(y=0; y<ordo; y++){
            scanf("%d", &isi);
            MatriksB[x][y]= isi;
        }
    }
    printf("Matriks AXB\n");
    for(x=0; x<ordo; x++){
        for(y=0; y<ordo; y++){
            for(z=0, hasil=0; z<ordo; z++){
                hasil+= MatriksA[x][z] * MatriksB[z][y];
            }
            printf("%d ", hasil);
        }
        printf("\n");
    }
}